Sources/VRage.Dedicated/Configurator/ConfigForm.cs

```diff
@@ -69,7 +69,7 @@ private void startButton_Click(object sender, EventArgs e)
             else // Local / Console
             {
                 string[] cmdLine = Environment.GetCommandLineArgs();
-                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? cmdLine[1] : "" ) + " -console -ignorelastsession");
+                Process.Start(cmdLine[0], ((cmdLine.Length > 1) ? String.Join(" ", cmdLine, 1, cmdLine.Length - 1 ) : "") + " -console -ignorelastsession");
```

Comment:
"Also, what happens if one of the cmdLine arguments is -console or -ignorelastsession? "
nothing. because this stage skipped...
"Can the started process handle duplicate command line arguments?" - why?

"According to ProcessStartInfo.Arguments arguments that contain strings need to be surrounded by quotes. And arguments containing quotes need to be escaped." - Ok. yep if it need to run properly. u can modifiy code by yourself. 

but, this is a quick bugfix, not a entire refactoring